### PR TITLE
feat(i18n): cache locale files

### PR DESCRIPTION
**Pull request by lidel**
This adds `i18next-localstorage-backend` which caches fetched translation files in `window.localStorage` for a week, which removes the need for seven requests for `.json` files.

ps. I also discovered that https://webui.ipfs.io/?lng=zh-CN works, so added it to README ;-)

### CODE REVIEW

**Comment by jessicaschilling** [comment]
Will this introduce a need to repeatedly clear caches when hacking on text in master `en` locale?

**Comment by lidel** [comment]
Good catch! Fixed in 09ba001b2bdff9c96ffb33540c184a4b5a2f53f2 –  disabled cache when developer server provided by `npm start` is used.

**Review by jessicaschilling — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -121,7 +123,7 @@
     "go-ipfs": "0.6.0",
     "http-proxy": "^1.18.1",
     "http-server": "^0.12.3",
-    "ipfs": "^0.48.1",
+    "ipfs": "0.50.2",
```
Comment (rafaelramalho19) [inline package.json:126]: is this intended in this PR? Just want to make sure we test everything so we don't problems from upgrading ipfs :)

**Review by rafaelramalho19 — APPROVED** [review]

**Comment by rafaelramalho19** [comment]
Don't forget to update ipld-explorer-components translation array to match the one on this PR 😄

File: package.json (hunk 1 above)
Comment (lidel) [reply package.json:126]: Good catch! js-ipfs is used only in e2e tests. 

Not related to this PR, but I noticed we were not using hard-coded version so fixed it and bumped to the latest version to ensure we are green with the latest version.

**Comment by lidel** [comment]
@rafaelramalho19 added it to  https://github.com/ipfs-shipyard/ipld-explorer-components/pull/281 :ok_hand:

**Pull request merged**